myTestYaml.yaml

```diff
@@ -0,0 +1,48 @@
+# apiVersion: v1
```

Comment:
It's better to put this into a test folder instead of root folder.  And usually we don't use names like "myXXX" :)

Reply:
Thank you for mentioning it. Will change!